AFQ/segmentation.py

```diff
@@ -695,6 +688,7 @@ def segment_afq(self, tg=None):
         # the ROIs. This order is ARBITRARY but CONSISTENT (going from ROI0
         # to ROI1).
         self.logger.info("Re-orienting streamlines to consistent directions")
+        dist_to_atlas = self.dist_to_atlas / vox_dim
```

Comment:
This used to be set above (old line 581) but that section got deleted, because endpoint dictionaries are handled differently, so i decided to move this line down to where it is more relevant.